src/domain/app/BrowserApp.tsx

```diff
@@ -33,7 +33,7 @@ const matomoInstance = createMatomoInstance({
 // Prevent non-production data from being submitted to Matomo
 // by pretending to require consent to process analytics data and never ask for it.
 // https://developer.matomo.org/guides/tracking-javascript-guide#step-1-require-consent
-if (import.meta.env.VITE_ENVIRONMENT !== 'production' && window?._paq) {
+if (!import.meta.env.PROD && window?._paq) {
```

Comment:
Now when I see the actual use cases, I think the VITE_ENVIRONMENT actually really had a reason to be there. Do we want the matomo to be in use in test and stage environments? Because this will make them available there. `import.meta.env.VITE_ENVIRONMENT !== 'production'` was explciitly for the production environment, because VITE_ENVIRONMENT is set to 'production' only in the prod env (kummilapset.hel.fi).

Reply:
We have in AppConfig 

```typescript
  static get matomoEnabled() {
    return Boolean(
      getEnvOrError(import.meta.env.VITE_MATOMO_ENABLED, 'VITE_MATOMO_ENABLED')
    );
  }
 ```